docs/components/segmented-control.md

```diff
@@ -118,3 +180,58 @@ export default defineComponent({
 })
 </script>
 ```
+
+## Theming
+
+| Variable | Purpose
+|:-------- |:-------
+| `--KSegmentTextColor` | Option text color
+| `--KSegmentSelectedBackground` | Option background color when selected
```

Comment:
Should we allow for customizing the unselected background color as well? `--KSegmentBackground`

Reply:
I thought about this but wasn't sure. Under the covers we are using `KButton`'s `outline` appearance which doesn't allow you to customize the background of the button. We could easily hijack a different appearance if we want to support this though.

Reply:
Added support for this 👍🏻